src/components/pages/Home/HomeContainer.js

```diff
@@ -42,7 +42,8 @@ function HomeContainer() {
           <Spin tip="Loading..." size="large"></Spin>
         </div>
       )}
-      {authState.isAuthenticated && userInfo && <RenderHomePage />}
+      {/* {!authState.isAuthenticated && !userInfo && <RenderHomePage />} */}
```

Comment:
lets not get commented out code into our main branch. Delete the code if it is no longer needed/wanted.